pkgs/top-level/all-packages.nix

```diff
@@ -18868,4 +18868,12 @@ with pkgs;
   biblioteca = callPackage ../by-name/bi/biblioteca/package.nix {
     webkitgtk = webkitgtk_6_0;
   };
+
+  upbound = callPackage ../by-name/up/upbound/package.nix {
```

Comment:
Using `pkgs/by-name` means that you don't need to add anything to this `all-packages.nix` file. Instead, you should have two directories with a `package.nix`:

* `pkgs/by-name/up/upbound`
* `pkgs/by-name/up/upbound-main`